### ISSUE TITLE
bug: Cannot read properties of null (reading 'split')

### ISSUE THREAD

**Issue by ubuntupunk** [tiddly-gittly/TiddlyGit-Desktop#146]
## Environment

TiddlyGit 0.6.0
Electron 15.2.0
linux 5.11.0-40-generic
Locale: en-GB

## Additional Context

```typescript
TypeError: Cannot read properties of null (reading 'split')
    at new ge (/usr/lib/tiddlygit/resources/app.asar.unpacked/.webpack/main/webpack:/TiddlyGit/node_modules/isomorphic-git/index.js:1477:30)
    at Function.from (/usr/lib/tiddlygit/resources/app.asar.unpacked/.webpack/main/webpack:/TiddlyGit/node_modules/isomorphic-git/index.js:1500:12)
    at Function.config [as get] (/usr/lib/tiddlygit/resources/app.asar.unpacked/.webpack/main/webpack:/TiddlyGit/node_modules/isomorphic-git/index.js:1616:22)
    at gitdir (/usr/lib/tiddlygit/resources/app.asar.unpacked/.webpack/main/webpack:/TiddlyGit/node_modules/isomorphic-git/index.js:10167:9)
    at Object.remotes [as listRemotes] (/usr/lib/tiddlygit/resources/app.asar.unpacked/.webpack/main/webpack:/TiddlyGit/node_modules/isomorphic-git/index.js:10198:31)
    at urlWithCredential (/usr/lib/tiddlygit/resources/app.asar.unpacked/.webpack/main/webpack:/TiddlyGit/node_modules/git-sync-js/dist/inspect.js:52:11)
    at NoNeedToSync (/usr/lib/tiddlygit/resources/app.asar.unpacked/.webpack/main/webpack:/TiddlyGit/node_modules/git-sync-js/dist/credential.js:5:37)
    at /usr/lib/tiddlygit/resources/app.asar.unpacked/.webpack/main/webpack:/TiddlyGit/node_modules/git-sync-js/dist/index.js:119:33
```

---

**Comment by linonetwo** [comment]
Reported https://github.com/isomorphic-git/isomorphic-git/issues/1433

**Comment by linonetwo** [comment]
Seems fixed in 0.6.4?
